ADAL/src/ui/mac/ADAuthenticationViewController.m

```diff
@@ -67,6 +68,8 @@ - (BOOL)loadView:(ADAuthenticationError * __autoreleasing *)error
     
     if (_webView)
     {
+        _embeddedWebview = YES;
```

Comment:
Can this function ever be called twice? (i.e. can the controller object be reused?) If there's nothing preventing it in the SDK, let's assume it can. So, we may want to have an Else clause setting _embeddedWebview to NO explicitly.